### PR TITLE
Add python-absl-py package to the Mariner 

### PR DESCRIPTION
**Pull request by rikenm1**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
One of the Python runtime requirements for TensorFlow. Imported python-absl-py package from Fedora 37.

###### Change Log  

- Change:  Imported python-absl-py package from Fedora

###### Does this affect the toolchain?  

**YES/NO**
NO

###### Test Methodology

- Pipeline build id: 246789

### CODE REVIEW

File: SPECS/python-absl-py/python-absl-py.spec [hunk 1]
```diff
@@ -0,0 +1,112 @@
+Name:           python-absl-py
+Version:        1.2.0
+Release:        2%{?dist}
```
Comment (neha170) [inline SPECS/python-absl-py/python-absl-py.spec:3]: Release can be 1

File: SPECS/python-absl-py/python-absl-py.spec [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+%files -n python3-absl-py -f %{pyproject_files}
+%doc AUTHORS
+%doc CHANGELOG.md
+%doc CONTRIBUTING.md
+%doc README.md
+%doc smoke_tests
+%license LICENSE
+
```
Comment (neha170) [inline SPECS/python-absl-py/python-absl-py.spec:72]: I'm not sure why the rpm doesn't have any other files besides doc and license

Comment (oliviacrain) [reply SPECS/python-absl-py/python-absl-py.spec:72]: It does have files- the `%pyproject_save_files` macro generates a list of files to install, which is passed to the `%files` macro above using the `-f` flag.

File: SPECS/python-absl-py/python-absl-py.rpmlintrc [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# These are not real spelling errors...
```
Comment (oliviacrain) [inline SPECS/python-absl-py/python-absl-py.rpmlintrc:1]: We don't use rpmlint- we can safely delete this file.

File: SPECS/python-absl-py/python-absl-py.spec [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+%changelog
+* Wed Oct 5 2022 Riken Maharjan <rmaharjan@microsoft.com> - 0.13.0-2
```
[the remark this answers is not in the document]
Comment (rikenm1) [reply SPECS/python-absl-py/python-absl-py.spec:75]: the changelog they have is very weird.  the version number doesn't even match the change log.  
For my changelog, I will just have changelog version 2.1.0-1

File: SPECS/python-absl-py/python-absl-py.rpmlintrc (hunk 3 above)
Comment (rikenm1) [reply SPECS/python-absl-py/python-absl-py.rpmlintrc]: deleted.

File: SPECS/python-absl-py/python-absl-py.spec (hunk 1 above)
Comment (rikenm1) [reply SPECS/python-absl-py/python-absl-py.spec]: release is not 1

**Review by neha170 — APPROVED** [review]
Please see what you can fix for the linter, thanks

**Pull request merged**